stack.yaml

```diff
@@ -28,7 +28,7 @@ packages:
   extra-dep: true
 - location:
     git: https://github.com/serokell/kademlia.git
-    commit: 92043c7e80e93aeb08212e8ce42c783edd9b2f80
+    commit: 0774fdb4d222115dda4379209e23c34fe5dec886 
```

Comment:
Ok, but whole situation with `kademlia` looks like bardaq anyway. When I wrote the previous message I didn't check it, but now I see that:
1. Its version consists of 4 numbers, but we use 3 numbers.
2. Master isn't even protected.
3. You don't pay attention to versions, last time version was changed in 2016: https://github.com/serokell/kademlia/commit/e9841449f183afd89999db70688dd580d3d793f7
4. As a consequence of (3) there are no tags and CSL currently depends just on some commit, not on some particular version. Of course it's better than depending on feature branch, but it's still bardaq.